homeassistant/components/matter/switch.py

```diff
@@ -61,6 +63,49 @@ def _update_from_device(self) -> None:
         )
 
 
+@dataclass(frozen=True)
+class MatterNumericSwitchEntityDescription(
```

Comment:
```suggestion
class MatterEnumSwitchEntityDescription(
```

small nit but this better reflects it

Reply:
This better reflects the case with enum backend. But should be usable for number backend as well.